### PR TITLE
Draft of cancellation announcement

### PR DESCRIPTION
**Pull request by swapnilogale**

### CODE REVIEW

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+:template: {{year}}/generic.html
+
+.. post:: August 25, 2021
+   :tags: {{shortcode}}-{{year}}, announcement, conference
+
+Cancelling WTD Australia and India 2021 conference
+===================================================
+
+G'day folks, we have decided to cancel our WTD Australia and India conference in 2021.
+While this hasn't been the easiest of decisions, we recognise the immense mental fatigue that lockdowns have had in Australia this year.
```
Comment (plaindocs) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
While this has been an extremely difficult decision, we recognise the immense mental fatigue that lockdowns have caused in Australia and India this year.
```

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Overall, while the community engagement has been fantastic across our remote meetup events, we are have all felt stressed and burnt out at some point these last couple of years.
```
Comment (plaindocs) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
While the community engagement and support have been fantastic across our remote meetup events and last year's conference, we are have all felt stressed and burnt-out at various times during last couple of years.
```

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Keeping in mind the long(er)-term health of the community, we have decided to do something else in place of the conference.
+
+Here's what we will do instead.
+
+Mega meetup
+------------
+
+In place of a full 2-day conference, we will select 4 talks (from the CFP process) and invite these documentarians to present their talks at a mega 2-hour meetup (date **TBA**).
```
Comment (plaindocs) [inline docs/conf/australia/2021/news/cancel-announcement.rst]: Do you want to be less specific about this before talking to speakers?

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Here's what we will do instead.
+
+Mega meetup
+------------
+
+In place of a full 2-day conference, we will select 4 talks (from the CFP process) and invite these documentarians to present their talks at a mega 2-hour meetup (date **TBA**). 
+We value the time and effort our documentarians put in to submit talk proposals and wanted to use this time to share their experience and knowledge.
+
+* This meetup event will be a non-ticketed event, so you can join in from anywhere, at any time. We will try and keep this in a timezone convenient for both our WTD Australia and India communities.
```
Comment (plaindocs) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
* This meetup event will be a non-ticketed event, so you can join in from anywhere. We will try and keep this in a timezone convenient for both our WTD Australia and India communities.
```

**Review by plaindocs — COMMENTED** [review]
I love the people-first tone of this. 

Take all of my suggested edits as optional, reword as needed.

@thatdocslady do you have further thoughts?

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 5]
```diff
@@ -0,0 +1,41 @@
+:template: {{year}}/generic.html
+
+.. post:: August 25, 2021
+   :tags: {{shortcode}}-{{year}}, announcement, conference
+
+Cancelling WTD Australia and India 2021 conference
+===================================================
+
+G'day folks, we have decided to cancel our WTD Australia and India conference in 2021.
```
Comment (thatdocslady) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
G'day folks. We're writing to let you know that we have decided to cancel our WTD Australia and India conference for 2021.
```

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+While this has been an extremely difficult decision, we recognise the immense mental fatigue that lockdowns have caused in Australia and India this year.
```
Comment (thatdocslady) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
This has been an extremely difficult decision, but we recognise the immense mental fatigue that the pandemic in general and lockdowns specifically have caused in Australia and India this year.
```

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+While the community engagement and support have been fantastic across our remote meetup events and last year's conference, we have all felt stressed and burnt-out at various times during the last couple of years.
```
Comment (thatdocslady) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
While the community engagement and support have been fantastic across our remote meetup events and last year's conference, we all felt stressed and burnt-out at various times during the last couple of years.
```

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+Keeping in mind the long(er)-term health of the community, we have decided to do something else in place of the conference.
```
Comment (thatdocslady) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
Keeping in mind the long(er)-term health of the community, we have decided to do something else in place of the conference, in a smaller scope that will be more manageable to the organizing team while still providing some continuity to the community. 
```

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+Here's what we will do instead.
+
+Mega meetup
```
Comment (thatdocslady) [inline docs/conf/australia/2021/news/cancel-announcement.rst]: Maybe "mega" isn't the word that we want to use externally? I don't have a better suggestion....

File: docs/conf/australia/2021/news/cancel-announcement.rst (hunk 3 above)
Comment (thatdocslady) [reply docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
In place of a full 2-day conference, we will select 4 talks (from the CFP process, pending confirmation from the speakers) and invite these documentarians to present their talks at a mega 2-hour meetup (date **TBA**). 
```

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+------------
+
+In place of a full 2-day conference, we will select 4 talks (from the CFP process) and invite these documentarians to present their talks at a mega 2-hour meetup (date **TBA**). 
+We value the time and effort our documentarians put in to submit talk proposals and wanted to use this time to share their experience and knowledge.
+
+* This meetup event will be a non-ticketed event, so you can join in from anywhere. We will try and keep this in a timezone convenient for both our WTD Australia and India communities.
```
Comment (thatdocslady) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
* This meetup event will be a free and open event without prior registration, so you can join in from anywhere. We will try and keep this in a timezone convenient for both our WTD Australia and India communities.
```

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+----------------
+
+2021 has been a great year with our remote meetup events in Australia.
+We hope you take this time to recover and refresh yourselves mentally and physically.
+We will be back in 2022 with the conference (onsite hopefully!).
+
+Some conference housekeeping
+-----------------------------
+
+* If you have already brought a ticket for the conference, we will refund these promptly.
```
Comment (thatdocslady) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
* If you already registered a ticket for the conference, we will refund these promptly and in full, without cancellation fees.
```

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+* If you had blocked these dates in your calendar, we will try and organise the mega meetup on one of these days, so that you do not have to take additional time off from work.
+* For those who have submitted a talk proposal, we thank you greatly and invite you to resubmit your talks for our conferences (Portland, Prague, Australia/India) next year.
```
Comment (thatdocslady) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
* For those who submitted a talk proposal, we thank you greatly and invite you to resubmit your talks for our conferences (Portland, Prague, Australia/India) next year.
```

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+* We will be closing the CFP process over the weekend, and also cancel future ticket sales and publish this announcement on the conference website, along with the relevant Write the Docs Slack channels and other social media.
```
Comment (thatdocslady) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
* We will be closing the CFP process before the weekend, cancel future ticket sales, and publish this announcement on the conference website, along with the relevant Write the Docs Slack channels and other social media.
```

**Review by thatdocslady — COMMENTED** [review]
Looks great, bunch of nit picks to be considered or ignored as you wish, Swapnil! Super proud of you and happy that we can support this <3

File: docs/conf/australia/2021/news/cancel-announcement.rst (hunk 9 above)
Comment (mxsasha) [reply docs/conf/australia/2021/news/cancel-announcement.rst]: Super?

File: docs/conf/australia/2021/news/cancel-announcement.rst [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+2021 has been a great year with our remote meetup events in Australia.
+We hope you take this time to recover and refresh yourselves mentally and physically.
+We will be back in 2022 with the conference (onsite hopefully!).
+
+Some conference housekeeping
+-----------------------------
+
+* If you already registered a ticket for the conference, we will refund these promptly and in full, without cancellation fees.
+* If you had blocked these dates in your calendar, we will try and organise the mega meetup on one of these days, so that you do not have to take additional time off from work.
+* For those who submitted a talk proposal, we thank you greatly and invite you to resubmit your talks for our conferences (Portland, Prague, Australia/India) next year.
```
Comment (mxsasha) [inline docs/conf/australia/2021/news/cancel-announcement.rst]:
```suggestion
* For those who submitted a talk proposal, we thank you greatly and invite you to resubmit your talks for our conferences (Portland, Prague, Australia/India) next year. If we have selected your talk for the mega meetup, we will contact you within the next two weeks.
```
(sticking with mega meetup term here for consistency, I agree with Mikey that "mega" is not my favourite term)

**Review by mxsasha — CHANGES_REQUESTED** [review]
I think this is great but needs an extra reassurance paragraph, like:
```
Other conferences and activities
--------------------------------
This announcement will not affect any of our other conferences and activities.
Our Prague 2021 virtual conference will still take place as planned. Tickets are
currently on sale, and we plan to announce the schedule soon. 
```

File: docs/conf/australia/2021/news/cancel-announcement.rst (hunk 9 above)
Comment (ericholscher) [reply docs/conf/australia/2021/news/cancel-announcement.rst]: I like the alliteration, but don't feel strongly about it.

**Review by ericholscher — APPROVED** [review]
I don't have much to add. I don't feel strongly about the naming of the event, but think the content reads nicely. 👍

File: docs/conf/australia/2021/news/cancel-announcement.rst (hunk 9 above)
Comment (swapnilogale) [reply docs/conf/australia/2021/news/cancel-announcement.rst]: I am ok with the word "Super". Let me change that.

**Comment by swapnilogale** [comment]
> I think this is great but needs an extra reassurance paragraph, like:
> 
> ```
> Other conferences and activities
> --------------------------------
> This announcement will not affect any of our other conferences and activities.
> Our Prague 2021 virtual conference will still take place as planned. Tickets are
> currently on sale, and we plan to announce the schedule soon. 
> ```

Have added this in @mxsasha

**Review by thatdocslady — APPROVED** [review]
All the edits look great, I think we can push this out today. I can buffer the social posts for it, and I assume that Mailchimp RSS will happen automatically?

**Comment by swapnilogale** [comment]
Thanks everyone for your support! Appreciate it.

**Review by mxsasha — APPROVED** [review]
Looks great, thanks!

**Comment by swapnilogale** [comment]
@thatdocslady happy for this to be merged?

**Pull request merged**